prbench-bilevel-planning/src/prbench_bilevel_planning/env_models/dynamic3d/tidybot3d_cupboard_real.py

```diff
@@ -49,13 +46,13 @@ def create_bilevel_planning_models(
     sim = ObjectCentricTidyBot3DEnv(
         scene_type="cupboard_real",
         num_objects=num_objects,
-        render_images=render_images,
+        render_images=False,
     )
 
     # State and goal abstractors.
     abstractor = CupboardRealStateAbstractor(sim)
     state_abstractor = abstractor.state_abstractor
-    goal_deriver = abstractor.goal_deriver_place
+    goal_deriver = abstractor.goal_deriver_place_four_cubes
```

Comment:
why do we need different goal derivers? can't the number of cubes be determined from the state?